flair/models/sequence_tagger_utils/biaffine.py

```diff
@@ -0,0 +1,156 @@
+        The         O      O          O        O      O       O    O      O        O     O        O     O      O       O    O      O       O   O
+        European           O         ORG       O      O       O    O      O        O     O        O     O      O       O    O      O       O   O
+        Commission                    O        O      O       O    O      O        O     O        O     O      O       O    O      O       O   O
+        said                                   O      O       O    O      O        O     O        O     O      O       O    O      O       O   O
+        on                                            O       O    O      O        O     O        O     O      O       O    O      O       O   O
+        Thursday                                              O    O      O        O     O        O     O      O       O    O      O       O   O
+        it                                                         O      O        O     O        O     O      O       O    O      O       O   O
+        disagreed                                                         O        O     O        O     O      O       O    O      O       O   O
+        with                                                                       O     O        O     O      O       O    O      O       O   O
+        German                                                                          MISC      O     O      O       O    O      O       O   O
+        advice                                                                                    O     O      O       O    O      O       O   O
+        to                                                                                              O      O       O    O      O       O   O
+        consumers                                                                                              O       O    O      O       O   O
+        to                                                                                                             O    O      O       O   O
+        shun                                                                                                                O      O       O   O
+        British                                                                                                                   MISC     O   O
+        lamb                                                                                                                               O   O
+        .                                                                                                                                      O
+        :return: 1-D  gold labels
+        """
+
+        gold_labels = []
+        for sentence in sentences:
+            ner = {(label.data_point.tokens[0].idx-1, label.data_point.tokens[-1].idx-1):label.value for label in sentences[0].get_labels("ner")}
```

Comment:
```suggestion
            ner = {(label.data_point.tokens[0].idx-1, label.data_point.tokens[-1].idx-1):label.value for label in sentence.get_labels("ner")}
```
I think this line contains a bug, which extracts labels of the first sentence for all sentences

Reply:
You are right. Thank you very much！